source/ota.c

```diff
@@ -910,7 +913,7 @@ static void dataHandlerCleanup( IngestResult_t result )
     }
 
     /* Let main application know of our result. */
-    otaAgent.OtaAppCallback( ( result == IngestResultFileComplete ) ? OtaJobEventActivate : OtaJobEventFail );
+    otaAgent.OtaAppCallback( ( result == IngestResultFileComplete ) ? OtaJobEventActivate : OtaJobEventFail, NULL );
```

Comment:
I'd prefer if this was with an if else statement.

But if we want to keep the turnery operator, then can you add another set of parenthesis around the turnery operator?

Reply:
I will help in readability and will be added in next PR.